### ISSUE TITLE
Error while starting application

### ISSUE THREAD

**Issue by ivn21** [SimpleMobileTools/Simple-Notes#434]
Thank you for a very usefull application, I've been using it for some time without problems. 
But after some time the application began to crash right after start. I think the problem is in big size of storing data (notes). May be removing data (through Android delete data for application) will solve this but there is useful information there. Unfortunatelly I found no way to retrieve data without root access or application running.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by ivn21** [comment]
I used the old one but I tried to update to the last version (6.8.0 (83)) with the same result.

**Comment by tibbi** [comment]
so it is crashing with 6.8.0 after you imported some file? Cant you create a widget to see the note texts?

**Comment by ivn21** [comment]
I haven't imported anything, it just started crashing suddenly. I opened the app and after the first screen appeared there was the system message that application will be closed. I looked at the size of the application data in android settings and saw that it is quite big (about 0,5 Gb). I think that if the program tries to use RAM for info that big my phone can't handle it.

**Comment by tibbi** [comment]
the app is not doing anything at all... So even if you reinstall the app it is misbehaving right from the beginning?

**Comment by ivn21** [comment]
yes

**Comment by tibbi** [comment]
dont you have a rooted device, or one modified in a similar way?

**Comment by ivn21** [comment]
unfortunately no

**Comment by tibbi** [comment]
find the app in your device settings and do a "Clear data" on it. It will remove your notes though so make sure you create some backup by creating a widget or so

**Comment by ivn21** [comment]
And how can I make backup? Is there a way to save notes?

**Comment by tibbi** [comment]
If you cannot open the app, then no. But you see the actual note content by creating a widget no?

**Comment by ivn21** [comment]
Unfortunately no, the app crashes before showing notes.

**Comment by tibbi** [comment]
even if you create a widget? What device and OS version do you have?

**Comment by ivn21** [comment]
I can see only main window if I click any button app crashes. I am using a no-brand smartphone with Android 8.1 Oreo.

**Comment by tibbi** [comment]
no-brand? dont you have such device specific issues in other apps too?

**Comment by ivn21** [comment]
I don't think it is a device specific issue. Earlier the app worked fine and behaviour of other apps didn't change.
